docs/standard/get-started.md

```diff
@@ -25,6 +25,6 @@ There are a number of ways to get started with .NET. Because .NET is a massive p
 
 * The [.NET Core Tutorials](../core/tutorials/index.md) detail a number of ways you can get started with .NET Core using your operating system and tooling of choice.
 
-## Get started using Docker on .NET Framework
+## Get started using Docker on .NET Core
```

Comment:
```suggestion
## Get started using .NET Core on Docker
```

Reply:
Great catch! @mikkelbu this is a good change.

Reply:
@Thraka Done :)